static/app/views/dashboardsV2/widgetCard/releaseWidgetQueries.tsx

```diff
@@ -294,38 +351,18 @@ class ReleaseWidgetQueries extends Component<Props, State> {
     let releaseCondition = '';
     const releasesArray: string[] = [];
     if (isCustomReleaseSorting) {
-      try {
-        const releases = await api.requestPromise(
-          `/organizations/${organization.slug}/releases/`,
-          {
-            method: 'GET',
-            data: {
-              sort: 'date',
-              project: projects,
-              per_page: 50,
-              environments,
-              summaryStatsPeriod: period,
-            },
-          }
-        );
-
-        if (releases.length) {
-          releaseCondition += '(release:' + releases[0].version;
-          releasesArray.push(releases[0].version);
-          for (let i = 1; i < releases.length; i++) {
-            releaseCondition += ' OR release:' + releases[i].version;
-            releasesArray.push(releases[i].version);
-          }
-          releaseCondition += ')';
+      if (releases && releases.length) {
+        releaseCondition += 'release:[' + releases[0].version;
+        releasesArray.push(releases[0].version);
+        for (let i = 1; i < releases.length; i++) {
+          releaseCondition += ',' + releases[i].version;
+          releasesArray.push(releases[i].version);
+        }
+        releaseCondition += ']';
```

Comment:
Its low chance we'll re-use this logic elsewhere right? Wondering if it'd be worth moving this to a util function